gulpfile.js

```diff
@@ -124,9 +124,17 @@ gulp.task("default", function (callback) {
     runSequence("clean", "compile", "tslint", callback);
 });
 
-gulp.task("test-ios", function (callback) {
+gulp.task("test-ios",["test-ios-uiwebkit", "test-ios-wkwebkit"]);
```

Comment:
The task name is wrong here: `test-ios-uiwebview`. Also, if I remember correctly, due to paths in tests we cannot run them in parallel so I would make them run in sequence like this:
```javascript
gulp.task("test-ios", function (callback) {
    runSequence("test-ios-uiwebview", "test-ios-wkwebview", callback);
});
```